includes/core-changes/corefx/3.0/fieldinfo-setvalue-exception.md

```diff
@@ -0,0 +1,38 @@
+### FieldInfo.SetValue throws exception for static, initialization-only fields
+
+Starting in .NET Core 3.0, an exception is thrown when you attempt to set a value on a static, initialization-only field by calling <xref:System.Reflection.FieldInfo.SetValue%2A?displayProperty=fullName>.
+
+#### Change description
+
+In .NET Framework and versions of .NET Core prior to 3.0, you could set the value of a static, initialization-only ([readonly in C#](~/docs/csharp/language-reference/keywords/readonly.md)) field by calling <xref:System.Reflection.FieldInfo.SetValue%2A?displayProperty=fullName>. However, setting a static, initialization-only field in this way resulted in unpredictable behavior based on the target framework and optimization settings.
+
+In .NET Core 3.0 and later versions, when you call <xref:System.Reflection.FieldInfo.SetValue%2A> on a static, initialization-only field, a <xref:System.FieldAccessException?displayProperty=nameWithType> exception is thrown.
+
+> [!TIP]
+> An initialization-only field is one that can only be set at the time it's declared or in the constructor for the containing class.
```

Comment:
This doesn't seem to fit the description of a tip ("Optional information to help a user be more successful"), and since the text above includes a clarification in parentheses, I would say it's not needed.

Reply:
I guess I disagree on this one. To me it is optional information for those that aren't already familiar with 'readonly'. Plus I don't think it harms anything by being included.